src/modules/Common/components/TextContent.tsx

```diff
@@ -18,6 +20,7 @@ const TextContent: React.FC<TextContentProps> = ({
       className={classNames(
         s.textContent,
         { [s.textContent__marginTopLarge]: !!marginTopLarge },
+        marginTop === false ? s.textContent__hasNoMarginTop : null,
```

Comment:
Use `!marginTop` in case undefined is passed

```
classNames(
        s.textContent,
        {
          [s.textContent__marginTopLarge]: marginTopLarge,
          [s.textContent__hasNoMarginTop]: !marginTop,
        },
        className
      )
```